src/main/java/com/gamebros/purepazaak/PurePazaak.java

```diff
@@ -22,22 +42,41 @@ public boolean closeRequested() {
   }
 
   public void render(GameContainer container, Graphics graphics) throws SlickException {
-    graphics.drawString("Pure Pazaak", 50, 50);
-
-    testCard.render(container, graphics);
-    testCard2.render(container, graphics);
+    this.matchController.render(graphics);
+    this.deckController.render(graphics);
+    this.uiController.render(graphics);
   }
 
   public void update(GameContainer container, int delta) {
-    return;
+    this.time += delta;
+
+    // Simulate time thinking.
```

Comment:
Unclear what this comment means.  `"Delay when the AI plays a card to slow the game down and feel more human-like."`